onions-impact.md

```diff
@@ -10,10 +10,15 @@ As the ONIONS [charter](./onions.md) is being discussed, here is an initial list
 
 
 ## I-Ds
-|WG|I-D|Title|Remarks|
+|WG|I-D|Title|Stay in Current WG or Move to ONIONS?|
```

Comment:
When I first saw the table, I thought what is M and S :-) .  My initial read was "medium" vs. "small" effort.  Maybe change this to Stay (S) in Current WG or Move (M) to ONIONS?